lib/main.coffee

```diff
@@ -96,10 +104,11 @@ module.exports =
         command = @command
         confFile = helpers.findFile(path.dirname(filePath), ['phpcs.xml', 'phpcs.ruleset.xml'])
         standard = if @autoConfigSearch and confFile then confFile else standard
+        legacy = @legacy
         return [] if @disableWhenNoConfigFile and not confFile
         if standard then parameters.push("--standard=#{standard}")
         parameters.push('--report=json')
-        text = 'phpcs_input_file: ' + filePath + eolChar + textEditor.getText()
+        if legacy then text = textEditor.getText() else text = 'phpcs_input_file: ' + filePath + eolChar + textEditor.getText()
```

Comment:
Can you split this up? The line is [too long](https://travis-ci.org/AtomLinter/linter-phpcs/builds/87399555#L106).

Maybe create a `prefix` var that legacy switches between nothing and the `'phpcs_input_file...` string?